### ISSUE TITLE
Pipeline capabilities scanner is reporting wrong name in scanner logs.

### ISSUE THREAD

**Issue by mohammedzee1000** [CentOS/container-pipeline-service#259]
We have reports that scanner logs of container capabilities scanner is giving the wrong container name.
I have verified this and here are some samples of erroneous logs.

Container name : registry.centos.org:fabric8/openshift-nginx-tag:1.0.1
Log File : https://registry.centos.org/pipeline-logs/NDBjZGZkMGIyOG/container-capabilities-results.json
Name shown : c2bf.rdu2.centos.org:5000/mattermost/mattermost-team:3.8

Container Name : registry.centos.org:eclipse/centos-python35:latest
Log file : https://registry.centos.org/pipeline-logs/NDBjZGZkMGIyOG/container-capabilities-results.json
Name shown : c2bf.rdu2.centos.org:5000/mattermost/mattermost-team:3.8
We should get this fixed ASAP.

**Comment by dharmit** [comment]
Issue is with the way we evaluate [`TEST_TAG`](https://github.com/CentOS/container-pipeline-service/blob/master/jenkinsbuilder/project-defaults.yml#L20).

Since for a weekly scan all the jobs get put onto the tube at almost same time, value for `TEST_TAG` for them is exactly same. This seems to be because same value till seconds for these jobs.

We can probably use `date +%N` instead to evaluate `TEST_TAG`. That ensures we get seconds since `1970-01-01 00:00:00 UTC` accurate up to nanoseconds.

**Comment by dharmit** [comment]
Before the fix was applied:

```
Image simple/wget sent for weekly scan with data {'logs_dir': '/srv/pipeline-logs/OGVkYmFhZGE1NT', 'tag': 'latest', 'output_image': 'dev-32-132.lon1.centos.org:5000/simple/wget:latest', 'notify_email': 'shahdharmit@gmail.com', 'TEST_TAG': 'OGVkYmFhZGE1NT', 'action': 'start_scan', 'weekly': True, 'namespace': 'simple-wget-latest', 'job_name': 'wget', 'image_under_test': 'registry.centos.org:simple/wget:latest'}
Image mattermost/mattermost-team sent for weekly scan with data {'logs_dir': '/srv/pipeline-logs/OGVkYmFhZGE1NT', 'tag': 3.8, 'output_image': 'dev-32-132.lon1.centos.org:5000/mattermost/mattermost-team:3.8', 'notify_email': 'shahdharmit@gmail.com', 'TEST_TAG': 'OGVkYmFhZGE1NT', 'action': 'start_scan', 'weekly': True, 'namespace': 'mattermost-mattermost-team-3.8', 'job_name': 'mattermost-team', 'image_under_test': 'registry.centos.org:mattermost/mattermost-team:3.8'}
```

After the fix was applied:

```
Image simple/wget sent for weekly scan with data {'logs_dir': '/srv/pipeline-logs/ZmQ2MDU5NmJjNT', 'tag': 'latest', 'output_image': 'dev-32-132.lon1.centos.org:5000/simple/wget:latest', 'notify_email': 'shahdharmit@gmail.com', 'TEST_TAG': 'ZmQ2MDU5NmJjNT', 'action': 'start_scan', 'weekly': True, 'namespace': 'simple-wget-latest', 'job_name': 'wget', 'image_under_test': 'registry.centos.org:simple/wget:latest'}
Image mattermost/mattermost-team sent for weekly scan with data {'logs_dir': '/srv/pipeline-logs/ZjU0OTY1MGQyMT', 'tag': 3.8, 'output_image': 'dev-32-132.lon1.centos.org:5000/mattermost/mattermost-team:3.8', 'notify_email': 'shahdharmit@gmail.com', 'TEST_TAG': 'ZjU0OTY1MGQyMT', 'action': 'start_scan', 'weekly': True, 'namespace': 'mattermost-mattermost-team-3.8', 'job_name': 'mattermost-team', 'image_under_test': 'registry.centos.org:mattermost/mattermost-team:3.8'}
```

Note the values of `TEST_TAG` key in the outputs.

**Comment by dharmit** [comment]
Closed via #262
